src/containers/sign-on/SignOnProvider.tsx

```diff
@@ -121,6 +121,7 @@ export class SignOnProvider extends Component<SignOnProps, SignOnState> {
       }
     })
 
+    this.props.fetchReferrer(this.props.location)
```

Comment:
Does this mean that the `ref` url query will only work for the /signin or /signup route? - just making sure that this is what we want?

Reply:
yep. it's just for referral to start a new account. what were you thinking?